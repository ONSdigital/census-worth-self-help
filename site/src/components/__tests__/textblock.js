import React from "react"
import renderer from "react-test-renderer"

import TextBlock from "../textblock"

describe("TextBlock", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<TextBlock>Text block content</TextBlock>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})