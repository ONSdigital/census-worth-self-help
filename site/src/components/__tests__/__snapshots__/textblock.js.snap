// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBlock renders correctly 1`] = `
<div
  className="Article-body-Style"
  css={
    Object {
      "map": undefined,
      "name": "pywoey",
      "next": undefined,
      "styles": "
        
    padding: 5px 15px;
  ;
      ",
    }
  }
>
  Text block content
</div>
`;
